lib/config/file.ts

```diff
@@ -30,5 +31,5 @@ export function getConfig(env: NodeJS.ProcessEnv): GlobalConfig {
     );
     config = migratedConfig;
   }
-  return config;
+  return massageConfig(migratedConfig);
```

Comment:
@rarkins I'm not sure if this is the right place. 🤔

Reply:
The alternative place would be here:
https://github.com/renovatebot/renovate/blob/89ad6401d931499fd253720bc7741a4afcba9d7a/lib/workers/repository/init/config.ts#L7-L9

Then the massage would also apply to the encrypted parts of the configuration.

Reply:
We need to make sure it won't message twice, thats my concern. Do not have the time to check this yet. sorry.